### DISCUSSION TITLE
How to set default value on relationship form field?

### DISCUSSION THREAD

**Discussion by Cnote43** [Budibase/budibase#10966] [Help] [answered]
I've been through the docs and found a few other related posts, but still can't figure out how to set a default value on a relationship form field. The default value isn't dependent on anything else. It will always be "Neutral", which is one of the options in the relationship column.

**Comment by ConorWebb96** [comment, marked as answer]
Hey @Cnote43,

This was easier to do in the past as the _id could be copied from a row in the data area. To solve this issue I outputted the value of the relationship picker into a paragraph component using the value of the relationship picker field. I then copied this _id and pasted it in the relationships picker's default value and it works as intended.

1. Add a paragraph inside the form. Bind it to the relationships field value.
<img width="1473" alt="Screenshot 2023-06-20 at 09 04 02" src="https://github.com/Budibase/budibase/assets/126772285/43ac1b8a-974e-4ebb-933c-10c1143a1b7d">
2. Select the chosen relationship you wish to use.
<img width="975" alt="Screenshot 2023-06-20 at 09 04 53" src="https://github.com/Budibase/budibase/assets/126772285/4159e6a9-e019-45f0-82e1-a758ed2b8df2">
3. _id should output to the paragraph. Copy this _id.
<img width="970" alt="Screenshot 2023-06-20 at 09 05 00" src="https://github.com/Budibase/budibase/assets/126772285/defba0ea-035e-44c5-9a86-e853bcb10b6f">
4.  Paste this _id into the relationship field's default value and hit save.
<img width="1477" alt="Screenshot 2023-06-20 at 09 05 49" src="https://github.com/Budibase/budibase/assets/126772285/1f25f189-3ecc-4e05-a60f-845c1030d41e">
5. Delete the paragraph and you should be good to go!

**See Result.**
![relationship defaulting](https://github.com/Budibase/budibase/assets/126772285/105a41bb-136f-468a-9e77-1426ff9e1107)

I hope this helps!

**Reply by Cnote43** [reply]
Thank you once again! Worked like a charm. Your product support is fantastic!
